prov/efa/src/rxr/rxr_attr.c

```diff
@@ -37,9 +37,16 @@
 const uint32_t rxr_poison_value = 0xdeadbeef;
 #endif
 
-#define RXR_TX_CAPS (OFI_TX_MSG_CAPS | FI_TAGGED | OFI_TX_RMA_CAPS | FI_ATOMIC)
+#ifdef HAVE_LIBCUDA
+#define EFA_HMEM_CAP FI_HMEM
+#else
+#define EFA_HMEM_CAP 0
+#endif
+#define RXR_TX_CAPS (OFI_TX_MSG_CAPS | FI_TAGGED | OFI_TX_RMA_CAPS | \
+		     FI_ATOMIC | EFA_HMEM_CAP)
 #define RXR_RX_CAPS (OFI_RX_MSG_CAPS | FI_TAGGED | OFI_RX_RMA_CAPS | \
-		     FI_SOURCE | FI_MULTI_RECV | FI_DIRECTED_RECV | FI_ATOMIC)
```

Comment:
Would you please add FI_ATOMIC back to TX and RX caps?

Reply:
It is still there on L#46 and L#49